web/client/components/map/openlayers/__tests__/Layer-test.jsx

```diff
@@ -1918,6 +1919,56 @@ describe('Openlayers layer', () => {
         expect(layer).toExist();
         expect(layer.layer.getSource().getParams().CQL_FILTER).toBe("prop = 'value'");
     });
+    it('test crossOrigin is applied to tiled wms', () => {
+        const options = {
+            type: "wms",
+            visibility: true,
+            name: "nurc:Arc_Sample",
+            group: "Meteo",
+            format: "image/png",
+            opacity: 1.0,
+            singleTile: false,
+            crossOrigin: "Anonymous",
+            url: "http://sample.server/geoserver/wms",
+            params: {
+                CQL_FILTER: "prop = 'value'"
+            }
+        };
+
+        let layer = ReactDOM.render(<OpenlayersLayer
+            type="wms"
+            options={options}
+            map={map}
+        />, document.getElementById("container"));
+
+        expect(layer).toExist();
+        expect(layer.layer.getSource().crossOrigin).toBe("Anonymous");
+    });
+    it('test crossOrigin is applied to single tile wms', () => {
+        const options = {
+            type: "wms",
+            visibility: true,
+            name: "nurc:Arc_Sample",
+            group: "Meteo",
+            format: "image/png",
+            opacity: 1.0,
+            singleTile: true,
+            crossOrigin: "Anonymous",
+            url: "http://sample.server/geoserver/wms",
+            params: {
+                CQL_FILTER: "prop = 'value'"
+            }
+        };
+
+        let layer = ReactDOM.render(<OpenlayersLayer
+            type="wms"
+            options={options}
+            map={map}
+        />, document.getElementById("container"));
+
+        expect(layer).toExist();
+        expect(layer.layer.getSource().crossOrigin_).toBe("Anonymous");
```

Comment:
maybe a typo?
```suggestion
        expect(layer.layer.getSource().crossOrigin).toBe("Anonymous");
```

Reply:
it's not a typo, in current ol the property have different names in TileWMS and ImageWMS